frontend/src/components/_common/InputGroup/Input/Input.tsx

```diff
@@ -0,0 +1,55 @@
+import { forwardRef, InputHTMLAttributes } from 'react';
+
+import { css } from 'styled-components';
+
+import type { InputStatus } from '@/components/_common/InputGroup/Input.type';
+
+import { theme } from '@/styles/theme';
+
+import * as S from './Input.styles';
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  status?: InputStatus;
+  $css?: ReturnType<typeof css>;
+  width?: string;
+  height?: string;
+  borderRadius?: string;
+  focusColor?: string;
+  onReset?: () => void;
```

Comment:
```suggestion
  $css?: ReturnType<typeof css>;
  status?: InputStatus;
  width?: string;
  height?: string;
  borderRadius?: string;
  focusColor?: string;
  onReset?: () => void;
```

`$css` 가 가장 위로 올라갔으면 좋겠어요! (가독성 문제)